### ISSUE TITLE
[Bug]: App does not show texts when primary OS language is different from english

### ISSUE THREAD

**Issue by mr-kew** [Whisky-App/Whisky#853]
### Description

I have Czech language set as primary, with english being secondary and by app looks like this: 
<img width="1012" alt="image" src="https://github.com/Whisky-App/Whisky/assets/26764516/c085d14f-a2c4-43fa-bc1f-c0e2eba35a92">

This affects all strings in the app.

If I set english as primary, everything works fine.

### Steps to reproduce

Set Czech language as primary language of the system. Leave english as secondary language if you want. Run Whisky.

### Expected behaviour

App should show texts in english, because czech localization is not available. Instead it shows the LocalizedKeys

### Logs

```bash
Whisky Version: 2.2.3
Date: 2024-02-29T18:55:43Z
macOS Version: 14.3.1

Arguments: --version

wine-7.7
```

### What version of Whisky are you using?

2.2.3

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by MaW-cmkbk** [comment]
I have same problem do this and it'll be fine. Its in Languange & locale.
![Snímek obrazovky 2024-03-03 v 9 44 37](https://github.com/Whisky-App/Whisky/assets/86100645/befc936d-1f6e-4cdc-acbb-cb0592b6c716)

**Comment by mr-kew** [comment]
Yeah, I also figured it out eventually, but still, it's a bug. Thanks anyway tho
